qforce/fragment.py

```diff
@@ -50,7 +53,7 @@ def reset_data_files(frag_dir):
             os.remove(data_path)
 
 
-def check_and_notify(job, config, n_unique, n_have):
+def check_and_notify(job, config, n_unique, n_have, n_generated):
```

Comment:
this function needs improvement, currently will not print correctly. I made some suggestions above and below to improve it:

n_missing = n_unique - n_have - n_generated